posts/2022-04-07-Rust-1.60.0.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Announcing Rust 1.60.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.60.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.60.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.60.0][notes] on GitHub.
+If you'd like to help us out by testing future releases, you might consider updating locally to use
+the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report] any bugs you might come across!
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1600-2022-04-07
+[report]: https://github.com/rust-lang/rust/issues/new/choose
+
+## What's in 1.60.0 stable
+
+### Instrumentation-based Code Coverage
```

Comment:
cc @tmandry and @richkadel 

When you have a minute, please take a look at this part of the release announcement. Thanks! 🙂

Reply:
The old version of coverage ("gcov") was also instrumentation based, but happened entirely in the LLVM backend using (lossy) debuginfo to map back to the source. This instrumentation is added by the language frontend before translation to LLVM IR and as a result is sometimes referred to as "source-based".

I talked about this some in the inside rust post first announcing the feature: https://blog.rust-lang.org/inside-rust/2020/11/12/source-based-code-coverage.html

Reply:
> This instrumentation is added by the language frontend before translation to LLVM IR and as a result is sometimes referred to as "source-based".

@tmandry - Just in case it's not clear (I had to re-read it before I realized what you were suggesting, I think), by "This instrumentation" you mean the new `-Cinstrument-coverage` (not "gcov"'s), and I think you are suggesting (and I agree) renaming the section:

```md
### Source-based Code Coverage
```

(which, FWIW, parallels clang's terminology for the same feature: https://clang.llvm.org/docs/SourceBasedCodeCoverage.html)